server/s3_get_object_action.cc

```diff
@@ -356,11 +367,12 @@ void S3GetObjectAction::set_total_blocks_to_read_from_fragmented_object() {
       extended_objects[first_byte_object_index].requested_object_size =
           (extended_objects[first_byte_object_index].object_size -
            (first_byte_offset_to_read -
-            extended_objects[first_byte_object_index].start_offset_in_object));
+            extended_objects[first_byte_object_index].start_offset_in_object) +
+           1);
 
       extended_objects[last_byte_object_index].requested_object_size =
           last_byte_offset_to_read -
-          (extended_objects[last_byte_object_index].start_offset_in_object);
+          (extended_objects[last_byte_object_index].start_offset_in_object) + 1;
```

Comment:
While testing 12MB multpart object, in specific scenario (with 1 motr_units_per_request), get range was returning one bytes less than the expected. This code change fixes that problem.